sklearn/metrics/scorer.py

```diff
@@ -514,6 +514,13 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 log_loss_scorer = make_scorer(log_loss, greater_is_better=False,
                               needs_proba=True)
 log_loss_scorer._deprecation_msg = deprecation_msg
+# Currently brier_score_loss don't support the shape of result
```

Comment:
@jnothman Thanks. I checked the document. roc_auc_score support shape = [n_samples] or [n_samples, n_classes]. (note that roc_auc_score belongs to _ThresholdScorer, and log_loss_score belongs to _ProbaScorer)